### ISSUE TITLE
Make TraCeR work with Trinity v2

### ISSUE THREAD

**Issue by mstubb** [Teichlab/tracer#3]
[Trinity version 2](https://github.com/trinityrnaseq/trinityrnaseq/wiki) uses different command line arguments to v1. It would be good if tracer could:

1. Use either version, depending on what's available and set in the config file.

2. Automatically determine the trinity version and call it correctly.

**Comment by rbagd** [comment]
Besides filenames, the only difference seems to be `--JM` in Trinity 1 and `--max_memory` in Trinity 2. Any ideas how to efficiently automate the selection?

**Comment by mstubb** [comment]
Hey @rbagd, 

Thanks for this. You should be able to do the following:

Get the version output with something like:

```
trinity = '/path/to/trinity' #either v1 or v2

try:
    subprocess.check_output([trinity, '--version'])

except subprocess.CalledProcessError as err:
    version_string = err.output
    # do things here to work out whether it's version 1 or 2 by parsing the version string.
    # it looks like the string from v2 will contain 'v2' whilst the v1 string will contain 
    # something like 'trinityrnaseq_r2014071'

```

**Comment by rbagd** [comment]
One annoying part is that `Trinity2 --version` runs an automatic network check for latest release but I guess that's a small price to pay.

**Comment by mstubb** [comment]
Yes, I think we can live with that.

**Comment by boombard** [comment]
Resolved by #5
